### PR TITLE
infra: remove url package duplication

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

**Review by trewest — APPROVED** [review]
/lgtm

File: tests/internal/url/url.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 
 	"github.com/cavaliergopher/grab/v3"
+
```
Comment (achuzhoy) [inline tests/internal/url/url.go:11]: redundant?

**Review by achuzhoy — APPROVED** [review]

**Review by yprokule — APPROVED** [review]
/lgtm

**Pull request merged**